components/nimbus/src/persistence.rs

```diff
@@ -1121,49 +1120,12 @@ mod tests {
         // force an upgrade & read in the upgraded database
         let db = Database::new(&tmp_dir).unwrap();
 
-        let db_experiments = db.collect_all::<Experiment>(StoreId::Experiments)?;
```

Comment:
@jhugman I took out the code in the roundtripping test that was doing the `json` -> `Experiment` -> `json` comparisons and instead made it validate that "we can still deserialize an old `json`" it doesn't validate that the Experiment is still correct though... I tried going `Experiment` -> `json` -> `Experiment` but I realized that the `Experiment` struct we have is a `v2` struct, so that flow doesn't really make sense for this test.